packages/mui-styled-engine/package.json

```diff
@@ -41,7 +41,17 @@
     "@babel/runtime": "^7.22.10",
     "@emotion/cache": "^11.11.0",
     "csstype": "^3.1.2",
-    "prop-types": "^15.8.1"
+    "prop-types": "^15.8.1",
+    "react": "^18.2.0"
```

Comment:
Are you sure that we need `prop-types` (!) and `react` (!!!) in `@mui/styled-engine` dependencies?

Reply:
cc @michaldudak